tests/InstrEngineTests/InstrEngineTests/ProfilerHelpers.cs

```diff
@@ -66,7 +77,7 @@ public static void LaunchAppUnderProfiler(string testApp, string testScript, str
             psi.EnvironmentVariables.Add("COR_ENABLE_PROFILING", "1");
             psi.EnvironmentVariables.Add("COR_PROFILER", ProfilerGuid.ToString("B"));
             psi.EnvironmentVariables.Add("COR_PROFILER_PATH", Path.Combine(PathUtils.GetAssetsPath(), string.Format("MicrosoftInstrumentationEngine_{0}.dll", bitnessSuffix)));
-            psi.EnvironmentVariables.Add("MicrosoftInstrumentationEngine_FileLog", "Dumps");
+            psi.EnvironmentVariables.Add("MicrosoftInstrumentationEngine_LogLevel", "Dumps");
```

Comment:
>MicrosoftInstrumentationEngine_LogLevel [](start = 42, length = 39)

Add another test that disables LogLevel altogether

Reply:
This requires adding a regex matching baseline (since I rely on the global logger for XML nodes)

---
In reply to: [344955335](https://github.com/microsoft/CLRInstrumentationEngine/pull/165#discussion_r344955335) [](ancestors = 344955335)